azure-pipelines.yml

```diff
@@ -1,44 +1,37 @@
-# Starter pipeline
-# Start with a minimal pipeline that you can customize to build and deploy your code.
-# Add steps that build, run tests, deploy, and more:
 # https://aka.ms/yaml
 
 trigger:
-  - main
+  - "*"
 
 pool:
   vmImage: "ubuntu-latest"
 
 variables:
-  SSH_HOST: "20.215.42.144"
-  SSH_USER: "azureuser"
-  REPO_URL: "https://github.com/maksnowak/FileSharingSystem"
-  WORK_DIR: "/home/azureuser/app"
+  - group: deploy-variables
+  - group: docker-registry-credentials
 
 stages:
-  - stage: Deploy
-    displayName: Deploy Application
-    condition: eq(variables['Build.SourceBranch'], 'refs/heads/main')
+  - stage: BuildAndTest
```

Comment:
If it builds the app and runs tests, it's gonna resolve #10, right?

Reply:
Yes but only for the go services, Vue.js should be also added in the future.

Reply:
Ok, I will assign it to myself then